src/figuresClasses.ts

```diff
@@ -1,19 +1,191 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
```

Comment:
Create a separate type for the color, to not repeat the same code several times 
```suggestion
  color: Color';
```